src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Sql/Sprocs/AddInstanceV6.sql

```diff
@@ -112,14 +112,24 @@ BEGIN
         AND StudyInstanceUid = @studyInstanceUid
 
     IF @@ROWCOUNT = 0
-    BEGIN
+    BEGIN TRY
```

Comment:
Do we need to worry about this happening when inserting into the `dbo.Series` table  too in case the `dbo.Study` row already existed but the `dbo.Series` row did not?

Reply:
Oh, never mind. The `UPDATE` on `dbo.Study` will force the transactions to execute serially so that there is never a problem adding the row into `dbo.Series`